config/locales/en.json

```diff
@@ -504,11 +504,18 @@
 	"dashboard.overall-health": "Patient's Overall Health",
 	"dashboard.expedited-processing": "Expedited processing for terminal and grave conditions",
 	"conditions.incomplete": "Not Completed",
+	"invite.expires": "This code will expire on %s",
+	"invite.code_loss_instruction": "If you lose your code, or it expires, fill out this form again to generate another code",
 	"invite.title": "Form submitted",
-	"invite.give_code": "To finish your application, give this code and link to your doctor or nurse practioner:",
+	"invite.give_code": "To finish your medical report, give this code to your doctor or nurse practitioner.",
```

Comment:
Is it possible to give this text more emphasis? The standard font-weight looks a bit high globally (to me). What font-weight do we use for bold?

Reply:
done. had just mis-spelled the font-semibold class.